### DISCUSSION TITLE
Year 2022, Season 3

### DISCUSSION THREAD

**Discussion by AllanChain** [AllanChain/micro-blog#4] [MicroBlog]
It's July now :)

**Comment by AllanChain** [comment]
中国人民爱好和平，论迹不论心，论心无日不好战。

理直气壮地幸灾乐祸也算是仇恨和对立之熏陶的累累硕果了。

**Comment by AllanChain** [comment]
**Physicist:**
- Release version 2.2 in 2005
- Release version 2.3 in 2006
- Release version 2.31 in 2007

**Developer:** That's a huge version bump in 2007.

**Physicist:** No, that's just some minor fixes.

**Comment by AllanChain** [comment]
七夕节不吉利啊，一年只见一次面，比异地还痛苦🐶

**Comment by AllanChain** [comment]
有些东西真是不干不净，吃了没命🐶

**Reply by AllanChain** [reply]
背景: 想起之前的霍乱病例和一些食品安全问题

**Comment by AllanChain** [comment]
_Various operating systems_

<img width="683" alt="image" src="https://user-images.githubusercontent.com/36528777/184045677-2d0d26fc-a868-4a02-af9d-2d2d9d2dc3ca.png">

Source: https://www.tendacn.com/us/product/W311MIAuto-Install.html

**Comment by AllanChain** [comment]
没有完美的人生，正如没有 bug free 的软件。但好消息是，虽然有 bug， 但大部分用户只需要 somehow 找到 workaround 就好了，不可能再回过头来细究这个 bug。

**Comment by AllanChain** [comment]
北大财务收费网站外包
1. 竟只有一个手机端
2. 竟在前端获取 IP 地址
3. 竟因为一个 ad blocker 无法正常付款
4. 竟没有关掉 source map
5. 竟把“请输入备注”写成了“请输入不备注”、把“勿删”写成了“误删”
6. 竟在注释里出现了神秘的“仙林学校”

<img width="944" alt="2022-08-25_22-05" src="https://user-images.githubusercontent.com/36528777/186686690-1695d13f-9a28-4af0-881e-76037c6a4003.png">

**Comment by AllanChain** [comment]
时间冲突定理：想选的通选课永远会与专业课的上课时间冲突，或者潜在冲突（如专业课掉课后选另一个班，可上可不上最好要上的专业课等）

**Comment by AllanChain** [comment]
You don't have a npy, even in the __future__:

<img width="355" alt="image" src="https://user-images.githubusercontent.com/36528777/187678177-46d9d3fe-ca5f-4030-8a0f-f310671870fa.png">

**Comment by AllanChain** [comment]
《现实残忍》&rarr;《孤寡一生》&rarr;《生殖隔离》&rarr;《画饼充饥》

<img width="577" alt="image" src="https://user-images.githubusercontent.com/36528777/187683467-a366b480-c827-4d6a-a2b6-390c827ec305.png">

**Reply by Yixuan-Wang** [reply]
![image](https://user-images.githubusercontent.com/21151119/187715357-2704375c-799a-4481-ab57-825b1a7c9f0c.png)

**Comment by AllanChain** [comment]
Is M$ pushing GitHub in the right direction?

https://github.com/community/community/discussions/31644#discussioncomment-3530579

<img width="394" alt="image" src="https://user-images.githubusercontent.com/36528777/188033828-dc886520-b31d-4972-b167-60b0207f89c3.png">

https://github.com/community/community/discussions/31644#discussioncomment-3529751

<img width="469" alt="image" src="https://user-images.githubusercontent.com/36528777/188034010-76f27068-a1c3-4abe-9280-cb5a88e63029.png">

**Comment by AllanChain** [comment]
人脑不是一个 consistent 的系统，你不可能让理智主持一切事物，总归会有一些所谓刻在 DNA 里的东西干扰行动。
代码不 consistent 了会出各种各样的问题，人脑也是这样。但代码好歹还能通过重构恢复 consistency，人脑又能由何方神圣重构呢？只能是 just works; don't touch 罢了。

**Reply by AllanChain** [reply]
背景: 道听途说 stories of brain inconsistency.

**Comment by AllanChain** [comment]
科学无国界，但科学家有国界。这或许算得上是科学家最大的悲哀吧。

就算你的理论能够超脱与一切传统框架，作为人还是得乖乖待在框架之中，动弹不得。

**Reply by Yixuan-Wang** [reply]
搞 natural science 还好一些🤪

**Comment by AllanChain** [comment]
关于所谓“窥屏”一事，不必苛责，它含有很多下意识的成分。且“人多眼杂”一说自古有之，密码不上屏历来如此。真有敏感内容就不要在人多的地方。这也不是说“窥屏”没有问题，当真无聊的时候还是看看自己的屏幕吧，盯着别人的屏幕看也不是什么礼貌的行为。

**Comment by AllanChain** [comment]
Move fast and break things. ❌

Move things and break fast. ✔️
